code/go/0chain.net/smartcontract/zcnsc/models.go

```diff
@@ -299,3 +307,17 @@ func (ps *poolStat) encode() []byte {
 	buff, _ := json.Marshal(ps)
 	return buff
 }
+
+type AuthCount struct {
+	Count int `json:"auth_count"`
+}
+
+func (ac *AuthCount) Encode() (data []byte, err error) {
+	data, err = json.Marshal(ac)
+	return
+}
+
+func (ac *AuthCount) Decode(input []byte) error {
+	err := json.Unmarshal(input, ac)
+	return err
```

Comment:
Could be one line code of `return json.Unmarhsal(input, ac)`

Reply:
did all these changes in #1682